### DISCUSSION TITLE
SQL schema

### DISCUSSION THREAD

**Discussion by RA-smile** [community/community#37791] [Other Feature Feedback, Questions, & Ideas]
I am new to coding and I was wondering if there is a website where you can copy and paste all the schema for SQL code and output a table (like an excel table with title, row, and column ) in order to visualize the schema clearer. Thank you. 

if you have other websites that made coding easier for you and are super helpful, please share. I dont have a community of "coders" so I am in this "coding world" alone lol.

**Comment by jfullstackdev** [comment]
frameworks are there to actually have everything setup for you and you just only focus on code, 

using PHP Laravel, you don't need to query raw MySQL but rather it's being handled by Laravel itself, 

getting, say, rows with same Sale ID will yield an Object that can easily be converted to Array and that array to table

but learning a framework is a steep learning curve, you need to learn PHP first and then the framework, you will feel overwhelmed but it's worth it

even if you already enter the Dev world, coding is really frustrating most of the time, it's a common feeling among all developers because it's really hard particularly if you are working in a codebase from others that are not maintainable in the long run

**Reply by RA-smile** [reply]
Thank you so much for your input!... I am currently looking into  PHP Laravel.
